src/qtiRunner/core/Renderer.js

```diff
@@ -544,18 +544,18 @@ const Renderer = function(options) {
                 requiredClasses = _.intersection(requiredClasses, _renderableClasses);
 
                 //add dependencies
-                _.each(requiredClasses, function(reqClass) {
+                _.each(requiredClasses, function (reqClass) {
```

Comment:
https://lodash.com/docs/4.17.15#forEach
```suggestion
                _.forEach(requiredClasses, function (reqClass) {
```